File: task-03/java/primen.java
import java.util.Scanner;
public class primen{
    public static void main(String[] args){
        Scanner sc= new Scanner(System.in);
        System.out.print("enter the no. :");
        int n= sc.nextInt();
        for(int i=1;i<n+1;i++){
            int f=0;
            for(int j=2;j<i;j++){
                if(i%j==0){
                    f=1;
                    break;
                }
            }
            if(f==0){
                System.out.println(i+": Prime");
            }
        }
        sc.close();

    }
}

File: task-03/javascript/primen.js
const readline = require('readline');
const rl=readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter a number: ", (input) => {
    const n = parseInt(input);
    for (let i = 1; i <= n; i++) {
        let f = 0;
        for (let j = 2; j<i; j++) {
            if (i % j == 0) {
                f = 1;
                break;
            }
        }
        if (f==0) {
            console.log(`${i} : Prime`);
        }
    }
    rl.close();


});